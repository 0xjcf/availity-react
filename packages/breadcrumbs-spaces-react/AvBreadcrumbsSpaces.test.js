import React from 'react';
import renderer from 'react-test-renderer';
import { AvBreadcrumbsSpaces } from '.';

describe('AvBreadcrumbsSpaces', () => {
  test('should render with attributes', () => {
    const component = renderer.create(
      <AvBreadcrumbsSpaces spaceId="1" spaceName="a" pageName="b" />
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });

  test('should render with with no name', () => {
    const component = renderer.create(
      <AvBreadcrumbsSpaces spaceId="1" pageName="b" />
    );
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
