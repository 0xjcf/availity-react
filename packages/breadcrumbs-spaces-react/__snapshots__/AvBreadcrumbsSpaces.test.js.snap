// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvBreadcrumbsSpaces should render with attributes 1`] = `
<ul
  className="breadcrumb"
>
  <li
    className="breadcrumb-item"
  >
    <a
      href="/public/apps/dashboard"
      target="newBody"
    >
      Home
    </a>
  </li>
  <li
    className="breadcrumb-item"
  >
    <a
      href="/public/apps/spaces/#/1"
    >
      a
    </a>
  </li>
  <li
    className="breadcrumb-item active"
  >
    b
  </li>
</ul>
`;

exports[`AvBreadcrumbsSpaces should render with with no name 1`] = `
<ul
  className="breadcrumb"
>
  <li
    className="breadcrumb-item"
  >
    <a
      href="/public/apps/dashboard"
      target="newBody"
    >
      Home
    </a>
  </li>
  <li
    className="breadcrumb-item"
  >
    <span>
      ···
    </span>
  </li>
  <li
    className="breadcrumb-item active"
  >
    b
  </li>
</ul>
`;
